rewrite-maven/src/main/java/org/openrewrite/maven/internal/MavenPomDownloader.java

```diff
@@ -208,6 +208,9 @@ private Pom getParentWithinProject(Pom projectPom, Map<Path, Pom> projectPoms) {
     }
 
     public MavenMetadata downloadMetadata(GroupArtifact groupArtifact, @Nullable ResolvedPom containingPom, List<MavenRepository> repositories) throws MavenDownloadingException {
+        if (containingPom != null) {
+            groupArtifact = containingPom.getValues(groupArtifact);
+        }
```

Comment:
I'm wondering if this block adds anything when the call below these lines to `downloadMetadata(GroupArtifactVersion gav, @Nullable ResolvedPom containingPom, List<MavenRepository> repositories) throws MavenDownloadingException` appears to do the same but for the GAV. Is there a reason to keep this in here and possibly get the values twice?
```suggestion
```